package.json

```diff
@@ -47,6 +47,7 @@
     "element-resize-detector": "^1.1.6",
     "lodash": "^4.11.0",
     "prop-types": "^15.6.0",
+    "react-addons-shallow-compare": "^15.6.2",
```

Comment:
nice!

Reply:
it's... actually kinda not nice, because it doesn't look like the package is maintained. But it's also temporary, once we move to react class syntax, we can just use `extend React.PureComponent` or whatever